netclient/functions/upgrades/v0.14.-6.go

```diff
@@ -0,0 +1,25 @@
+package upgrades
+
+import (
+	"github.com/gravitl/netmaker/logger"
+	"github.com/gravitl/netmaker/netclient/config"
+)
+
+var upgrade0146 = UpgradeInfo{
+	RequiredVersions: []string{
+		"v0.14.0",
+		"v0.14.1",
+		"v0.14.2",
+		"v0.14.3",
+		"v0.14.4",
+		"v0.14.5",
```

Comment:
this should be the only required version

Reply:
also the file name has a "-" in it, is it necessary?

Reply:
file name updated to match v0.-14-5.go

Reply:
> 

if someone is updating from an earlier version,  all the intermediate updates are needed.